deploy/gk-deploy

```diff
@@ -485,7 +503,7 @@ if [[ "${hello}" != "Hello from Heketi" ]]; then
 fi
 
 load_temp=$(mktemp)
-eval_output "heketi-cli -s http://${heketi_service} topology load --json=${TOPOLOGY} 2>&1" | tee ${load_temp}
+eval_output "heketi-cli -s http://${heketi_service} --user admin --secret "${ADMIN_KEY}" topology load --json=${TOPOLOGY} 2>&1" | tee ${load_temp}
```

Comment:
Is specifying the admin only required if an ADMIN_KEY is provided? If so, I imagine there's no problem specifying admin with no key?